core/templates/base-components/oppia-root.directive.ts

```diff
@@ -32,7 +32,7 @@ angular.module('oppia').directive('oppiaRoot', [
       controller: ['$scope',
         function($scope) {
           $scope.initialized = false;
-
+          OppiaAngularRootComponent.rteHelperService = RteHelperService;
```

Comment:
Why is this needed?

Reply:
RteHelperService hasn't been migrated yet. So I am storing a reference to that service (this is needed for CKEditor Widget Initialiser). Migrating RTE helper service is a pretty huge migration, we are still migrating the other stuff required for an easy migration of RTE helper service.

Reply:
Thanks!